packages/codegen-ui-react/lib/react-studio-template-renderer-helper.ts

```diff
@@ -186,3 +184,22 @@ export function bindingPropertyUsesHook(
 ): boolean {
   return isDataPropertyBinding(binding) && 'predicate' in binding.bindingProperties;
 }
+
+// optional import prettier
+export function formatCode(code: string): string {
+  let prettier = null;
+  let parserTypescript = null;
```

Comment:
You can get rid of the `let's` here if you wrap the return inside your `try` block below, and may clean up the code a bit.

Reply:
Good idea.